### ISSUE TITLE
Create random pedestrians

### ISSUE THREAD

**Issue by gizem-buss** [lgsvl/simulator#1855]
Hello,
I can't create random pedestrians on own map. I followed all the steps in the documentation but;
1) Pedestrians crossing the street
2) Pedestrians walking randomly on the road

I can't create the above scenarios. Can you help me?

![navmesh](https://user-images.githubusercontent.com/71923140/140296799-0fd56251-bd0f-4840-8024-9ab2d7c8357b.PNG)
![pedestrianpath](https://user-images.githubusercontent.com/71923140/140296820-a8917a55-bcc4-4601-9ec8-cbd8fd25f573.PNG)

**Comment by EricBoiseLGSVL** [comment]
Yes, @ertauren is correct, you can set a lower navigation area point value (higher value == pedestrian logic less likely to use) but I would also look closer to the curb.  If the nav mesh is broken, then agents can't get over or down to the road nav mesh.  See here:
![image](https://user-images.githubusercontent.com/45106230/140392682-5e00c1ce-a609-4cc4-aa65-0f6a28ee5619.png)

Another issue could be the check for the nav mesh we do.  Unity doesn't have apis to see if a nav mesh is present so we raycast at 0,0,0 in world space to see if a nav mesh is hit.  Make sure the nav mesh is created at this point or change the code to the point that fits your new map.
![image](https://user-images.githubusercontent.com/45106230/140392761-6845003e-fd35-46bc-aaed-e9e9005aae3a.png)

Lastly, please show how you organized your pedestrian annotation under the mapholder object.  Screen shot the Hierarchy panel.
![image](https://user-images.githubusercontent.com/45106230/140392832-570ca311-d5c1-40e3-9f6a-8ffaeed34b74.png)

Finally, be sure you have pedestrians in the AssetBundle folder in the editor as well as binary builds.  We provide compatible bundles in the binary and you just need to copy and place in the editor.  You should see logs in the Console panel if simulator detects no pedestrians to load or other issues.  Be sure to always look at and post console/playerlogs when posting issues to quickly see what the problem is.
![image](https://user-images.githubusercontent.com/45106230/140392931-2c93fb76-7a71-497a-9c6c-f534c0dd6eb9.png)
